src/main/java/io/kestra/storage/azure/AzureStorage.java

```diff
@@ -229,7 +229,7 @@ public URI move(String tenantId, URI from, URI to) throws IOException {
             mkdirs(dest);
             BlobClient sourceClient = client.getBlobClient(itemResult.getName());
             SyncPoller<BlobCopyInfo, Void> poller = client.getBlobClient(destName).beginCopy(sourceClient.getBlobUrl(),
-                Duration.ofSeconds(30));
+                Duration.ofSeconds(1));
```

Comment:
I changed this because it was causing the move operation to take "at least" 30 seconds since this param is not a timeout but a polling interval.
Btw 1 second is already the default polling interval but I find it better to explicitly say it :+1: